migration/migrator/migrations/course/20190608032611_grade_inquiry_with_timezone.py

```diff
@@ -0,0 +1,13 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    database.execute("ALTER TABLE regrade_discussion ALTER COLUMN timestamp TYPE TIMESTAMP WITH TIME ZONE;")
+    database.execute("ALTER TABLE regrade_requests ALTER COLUMN timestamp TYPE TIMESTAMP WITH TIME ZONE;")
+    pass
```

Comment:
I don't think a `pass` is needed here both in `up` and `down` functions - It is mostly used as a placeholder for future code, or just to make things syntactically correct.

> pass is a null operation — when it is executed, nothing happens. It is useful as a placeholder when a statement is required syntactically, but no code needs to be executed

https://docs.python.org/3.2/reference/simple_stmts.html#the-pass-statement